src/main/java/ch/njol/skript/log/LogEntry.java

```diff
@@ -159,15 +157,20 @@ public String toFormattedString() {
 			details = OTHER_DETAILS;
 		}
 
-		String from = this.from;
-		if (!from.isEmpty())
-			from = "§7   " + from + "\n";
-
 		// Replace configured messages chat styles without user variables
 		String lineInfoMsg = replaceNewline(Utils.replaceEnglishChatStyles(lineInfo.getValue() == null ? lineInfo.key : lineInfo.getValue()));
 		String detailsMsg = replaceNewline(Utils.replaceEnglishChatStyles(details.getValue() == null ? details.key : details.getValue()));
 		String lineDetailsMsg = replaceNewline(Utils.replaceEnglishChatStyles(LINE_DETAILS.getValue() == null ? LINE_DETAILS.key : LINE_DETAILS.getValue()));
 
+		if (node == null)
+			return String.format(detailsMsg.replaceAll("(\\s){2,}", ""), message); // Remove line beginning spaces
```

Comment:
Not sure if replacing 2+ whitespace is the best approach here.

Also the group isn't needed I think: `\\s{2,}`

Reply:
Adding upon what TP said, you can use `String#stripLeading` to remove the leading whitespaces in a string

Reply:
`String#stripLeading` Java 12 feature :(